wdl-analysis/CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+* Refactored the `AnalysisResult` and `Document` types to move properties of
+  the former into the latter; this will assist in evaluation of documents in
+  that the `Document` along can be passed into evaluation ([#265](https://github.com/stjude-rust-labs/wdl/pull/265)).
```

Comment:
GH is hiding the `suggestion` button from me, but I think `along` should be `alone`

Reply:
```suggestion
  that the `Document` alone can be passed into evaluation ([#265](https://github.com/stjude-rust-labs/wdl/pull/265)).
```
Not sure why I couldn't do this before